change/office-ui-fabric-react-2019-06-20-19-36-43.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Adding new props for Sticky and ScrollablePane",
```

Comment:
You'll want to describe the _new behavior_ you are adding to `ScrollablePane` and `Sticky`. Reminder that the consumers of the library are the audience.